.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+repos:
+  # python linter and formatter
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    # Ruff version.
```

Comment:
does ruff use the black formatting style?

Reply:
seems like it [docs](https://github.com/astral-sh/ruff/blob/main/docs/faq.md#)